core/backend-itwin-client/src/RequestHost.ts

```diff
@@ -2,69 +2,90 @@
 * Copyright (c) Bentley Systems, Incorporated. All rights reserved.
 * See LICENSE.md in the project root for license terms and full copyright notice.
 *--------------------------------------------------------------------------------------------*/
+import { RequestGlobalOptions } from "@bentley/itwin-client";
 import * as http from "http";
 import * as https from "https";
+import { HttpsProxyAgent, HttpsProxyAgentOptions } from "https-proxy-agent";
 import * as url from "url";
-import { RequestGlobalOptions } from "@bentley/itwin-client";
 
-import HttpsProxyAgent = require("https-proxy-agent");
 /** Utility to configure all HTTP service requests make from the backend
  * @internal
  */
 export class RequestHost {
+  private static _proxyUrl?: string;

```

Comment:
This is a terrible name for a class, because "request" is usually a verb. Can it be HttpRequestHost, maybe?

Reply:
We have a class called Request that in retrospect could've been named HttpRequest. There are several other classes around that also - RequestGlobalOptions, etc. Perhaps we can rename all those to be consistent in 3.0?

Reply:
Renamed now.